corehq/apps/app_manager/static/app_manager/js/app-notifications.js

```diff
@@ -0,0 +1,24 @@
+/* globals hqDefine moment */
+hqDefine('app_manager/js/app-notifications.js', function () {
+    function NotifyFunction(userId, $element) {
+        return function(msg) {
+            var msgObj = JSON.parse(msg);
+            // only show notifcations from other users
+            if (msgObj.user_id !== userId) {
+                var $inner;
+                // if nothing in the outer element, add the notification div and populate
```

Comment:
Hrm, yeah, it is uglier, I'd just been thinking of it from the code perspective.

It would actually be nice for `alert_user` to have an option to update an existing message rather than create a new one (the multiple alerts thing happens occasionally on other pages, and it's basically always ugly), and doesn't look like it'd be hard, but I haven't thought it all the way through, so understand if you don't want to deal with that in this PR.